### PR TITLE
Add rootfs for Linux Test Project (LTP)

### PR DESCRIPTION
**Pull request by Lakshmipathi**
Include build and install script for LTP and update rootfs-config.yaml

Signed-off-by: Lakshmipathi <lakshmipathi.ganapathi@collabora.com>

### CODE REVIEW

**Comment by Lakshmipathi** [comment]
rootfs job: https://bot.staging.kernelci.org/job/rootfs-builder/135/

**Comment by gctucker** [comment]
I think the first step is to produce a rootfs and have it manually tested in a LAVA lab, to show it can boot and has the test binaries installed.  Then the next step is to add a job template, it may be done in a separate PR.

File: rootfs-configs.yaml [hunk 1]
```diff
@@ -110,7 +110,6 @@ rootfs_configs:
       - libext2fs2
     script: "scripts/buster-cros-ec-tests.sh"
     test_overlay: ""
-
```
Comment (gctucker) [inline rootfs-configs.yaml:113]: Please don't remove blank lines unnecessarily.  It causes noise in the diffs which can lead to git conflicts, and also blank lines help with readability of the file.

**Comment by Lakshmipathi** [comment]
This patch is for rootfs creation, in the process of creating test templates, which will be send as another PR.

**Comment by gctucker** [comment]
@Lakshmipathi please change the commit title to `rootfs-configs.yaml: add debian-ltp` to be clear that this commit and this PR only adds the rootfs images.

**Comment by Lakshmipathi** [comment]
okay changed title. Tested with `arm64` https://bot.staging.kernelci.org/job/rootfs-builder/143/

**Comment by gctucker** [comment]
Great, thanks.  Can you please just run a LAVA job that will boot it using nfsroot and list the contents of the directory where the LTP binaries are installed?

That should be enough to check this rootfs-only PR is valid.  Then the next PR will be about adding the test plan with actual test cases and all that.

**Comment by Lakshmipathi** [comment]
Ok, I ran this job yesterday: https://lava.collabora.co.uk/scheduler/job/2553971/definition It ran a single syscall test.  Does it look good?

**Review by gctucker — APPROVED** [review]
Yes, that looks like a good nfsroot boot test.  There were a couple of minor things though in the LTP test case run, not sure they're blockers:
```
/opt/ltp/ver_linux: 115: /opt/ltp/ver_linux: ethtool: not found
```
```
/opt/ltp/ver_linux: 135: /opt/ltp/ver_linux: free: not found
```
I think those can also be addressed in a follow-up as we're likely to be missing other subtle dependencies in the rootfs and we'll find out when running more tests.

**Pull request merged**